packages/perspective-phosphor/src/ts/widget.ts

```diff
@@ -44,6 +44,7 @@ export type PerspectiveWidgetOptions = {
     key?: string;
     wrap?: boolean;
     delete_?: boolean;
+    view?: any;
```

Comment:
Note that it is also possible to `extend` `PerspectiveWidget` adding the `view` options similar to how `JupyterPhosphorAccordionWidget` does this in `@jupyter-widgets/controls`.